apache-couchdb-mixin/dashboards/couchdb-overview.libsonnet

```diff
@@ -0,0 +1,1817 @@
+
+local dashboardUid = 'couchdb-overview';
+
+local promDatasourceName = 'prometheus_datasource';
+local matcher = 'job=~"$job", cluster=~"$cluster"';
+
+local promDatasource = {
+  uid: '${%s}' % promDatasourceName,
+};
+
+local numberOfClustersPanel = {
+  datasource: promDatasource,
+  targets: [
+    prometheus.target(
+      'count(count by(cluster, job) (couchdb_request_time_seconds_count{' + matcher + '}))',
+      datasource=promDatasource,
+      legendFormat='cluster="{{ cluster }}"',
+      format='time_series',
+    ),
+  ],
+  type: 'stat',
+  title: 'Number of clusters',
+  description: 'The number of clusters being reported.',
+  fieldConfig: {
+    defaults: {
+      color: {
+        mode: 'thresholds',
+      },
+      mappings: [],
+      thresholds: {
+        mode: 'absolute',
+        steps: [
+          {
+            color: 'yellow',
+            value: null,
+          },
+          {
+            color: 'red',
+            value: 0,
+          },
+          {
+            color: 'green',
+            value: 1,
+          },
+        ],
+      },
+    },
+    overrides: [],
+  },
+  options: {
+    colorMode: 'value',
+    graphMode: 'none',
+    justifyMode: 'auto',
+    orientation: 'auto',
+    reduceOptions: {
+      calcs: [
+        'lastNotNull',
+      ],
+      fields: '',
+      values: false,
+    },
+    textMode: 'auto',
+  },
+  pluginVersion: '9.2.3',
+};
+
+local numberOfNodesPanel = {
+  datasource: promDatasource,
+  targets: [
+    prometheus.target(
+      'sum(count by(cluster, job) (couchdb_request_time_seconds_count{' + matcher + '}))',
+      datasource=promDatasource,
+      legendFormat='cluster="{{ cluster }}"',
+      format='time_series',
+    ),
+  ],
+  type: 'stat',
+  title: 'Number of nodes',
+  description: 'The number of nodes being reported.',
+  fieldConfig: {
+    defaults: {
+      color: {
+        mode: 'thresholds',
+      },
+      mappings: [],
+      thresholds: {
+        mode: 'absolute',
+        steps: [
+          {
+            color: 'red',
+            value: null,
+          },
+          {
+            color: 'red',
+            value: 0,
+          },
+          {
+            color: 'green',
+            value: 1,
+          },
+        ],
+      },
+    },
+    overrides: [],
+  },
+  options: {
+    colorMode: 'value',
+    graphMode: 'none',
+    justifyMode: 'auto',
+    orientation: 'auto',
+    reduceOptions: {
+      calcs: [
+        'lastNotNull',
+      ],
+      fields: '',
+      values: false,
+    },
+    textMode: 'auto',
+  },
+  pluginVersion: '9.2.3',
+};
+
+local clusterHealthPanel = {
+  datasource: promDatasource,
+  targets: [
+    prometheus.target(
+      'min(min by(cluster, job) (couchdb_couch_replicator_cluster_is_stable{' + matcher + '}))',
+      datasource=promDatasource,
+      legendFormat='cluster="{{ cluster }}"',
+      format='time_series',
+    ),
+  ],
+  type: 'stat',
+  title: 'Cluster health',
```

Comment:
How will this react/work in a multi-cluster setup? Multiple Healthy/unhealthy signs?
If so, how do you relate the health status to a specific cluster other than using the label selectors at the top?

Reply:
Bump on this comment

Reply:
@Dasomeone I made some changes here, so let me know what you think.

Reply:
How about we show **Percentage of Clusters healthy** instead? That could still be **green** (when 100%) and **red** when it is less than 100%. What do you think @Dasomeone 
I don't think the green panel with cluster1 is visually appealing.

Reply:
Agreed, percentage of clusters, or num/total_clusters both are good solutions as well